tests/unit/utils/test_otp.py

```diff
@@ -34,7 +34,7 @@ def test_generate_totp_secret():
     secret = generate_totp_secret()
 
     # secret should be bytes
-    assert type(secret) == bytes
+    assert type(secret) is bytes
```

Comment:
![bill-clinton-meaning-of-the-word-is](https://github.com/pypi/warehouse/assets/529516/b81fb9b9-25e9-4a52-a19c-3a7a01b04574)